### ISSUE TITLE
Remove slider attributes logging

### ISSUE THREAD

**Issue by cdtinney** [brownieboy/react-bootstrap-slider#30]
It would be nice if the sliderAttributes logging that's done on creation could be removed. It causes some spam when testing and debugging.

It's in `dist\react-bootstrap-slider.js` on line 122.

`console.log("sliderAttributes = " + JSON.stringify(sliderAttributes, null, 4));`

It produces output like...

``` 
sliderAttributes = {
          "id": "size-slider",
          "value": 1,
          "step": 1,
          "max": 5,
          "min": 1,
          "tooltip": "show"
      }
```

**Comment by saevarom** [comment]
#31 fixes this

**Comment by brownieboy** [comment]
Closed by #31 merge
